data/strings/types_strings.txt

```diff
@@ -4835,7 +4835,7 @@
     fi = Stripclub
     fr = Club de strip-tease
     he = מועדון חשפנות
-    hu = Stripclub
+    hu = Striptízklub
```

Comment:
Typo, correct form is "Sztriptízklub" but I'd rather use the term "Sztriptízbár".